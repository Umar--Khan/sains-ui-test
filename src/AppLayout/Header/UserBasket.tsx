import { headerTestIds } from 'src/testUtils/const/testIds';

interface UserBasketProps {
  itemsInBasket: number;
}

const UserBasket = ({ itemsInBasket }: UserBasketProps) => (
  <span data-testid={headerTestIds.numberInBasket}>
    Basket ({itemsInBasket} items)
  </span>
);

export default UserBasket;
