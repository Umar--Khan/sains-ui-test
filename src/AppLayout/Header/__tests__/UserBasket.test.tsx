import renderer from 'react-test-renderer';

import UserBasket from '../UserBasket';

describe('<UserBasket />', () => {
  it('Should render correctly', () => {
    const component = renderer.create(<UserBasket itemsInBasket={2} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
