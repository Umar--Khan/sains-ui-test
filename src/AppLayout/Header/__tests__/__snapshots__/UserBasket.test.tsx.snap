// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserBasket /> Should render correctly 1`] = `
<span
  data-testid="number-in-basket"
>
  Basket (
  2
   items)
</span>
`;
